dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/datasource-mysql/docker-compose.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: "2.1"
+
+services:
+  dolphinscheduler:
+    image: apache/dolphinscheduler-standalone-server:ci
+    environment:
+      MASTER_MAX_CPU_LOAD_AVG: 100
+      WORKER_TENANT_AUTO_CREATE: 'true'
+    expose:
+      - 12345
+    networks:
+      - e2e
+    volumes:
+      - ./mysql-connector-java-8.0.16.jar:/opt/dolphinscheduler/libs/mysql-connector-java-8.0.16.jar
```

Comment:
You put the `.jar` file in the codebase, but we cannot include this binary file in our releases (due to release policy and license issue), you have 2 choices:

1. modify our release script to exclude this binary file, but that causes E2E tests failed if users run the tests from our release.
2. write a script to download the binary on the fly, like what we did in SkyWalking,

https://github.com/apache/skywalking/blob/master/test/e2e-v2/script/prepare/setup-oap/download-mysql.sh

https://github.com/apache/skywalking/blob/1eacba86fd60e5461b8d45e6a49a6e6636b3b1e1/test/e2e-v2/script/docker-compose/base-compose.yml#L29

https://github.com/apache/skywalking/blob/1eacba86fd60e5461b8d45e6a49a6e6636b3b1e1/test/e2e-v2/cases/storage/mysql/docker-compose.yml#L44